sass/sass.bzl

```diff
@@ -11,219 +11,170 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"Compile Sass files to CSS"
 
-SASS_FILETYPES = FileType([
-    ".sass",
-    ".scss",
-])
+_FILETYPES = [".sass", ".scss", ".svg", ".png", ".gif"]
 
-def collect_transitive_sources(ctx):
-    source_files = depset(order="postorder")
-    for dep in ctx.attr.deps:
-        source_files += dep.transitive_sass_files
-    return source_files
+# Documentation for switching which compiler is used
+_COMPILER_ATTR_DOC = """Choose which Sass compiler binary to use.
+
+By default, we use the JavaScript-transpiled version of the
+dart-sass library, based on https://github.com/sass/dart-sass.
+This is the canonical compiler under active development by the Sass team.
+This compiler is convenient for frontend developers since it's released
+as JavaScript and can run natively in NodeJS without being locally built.
+However, it is the slowest option. If you have a substantial Sass
+codebase, consider the two other options for this attribute:
+
+1. `compiler = "@sassc"` uses the libsass compiler written in C++.
+   As of 2018, this is the most commonly used compiler.
+   It requires your Bazel setup has a working C++ compilation toolchain.
+
+   NOTE: future releases of rules_sass may remove this option, if it
```

Comment:
Unless we have someone who's volunteering to actively maintain LibSass support, I think we should bite the bullet and drop it immediately, especially if support is going to require extra work to make command-line flags line up properly.

Reply:
For now it looks like `sourcemap` is the only one supported that's problematic. How about leaving it for now, and dropping it if we run into more mismatches?

Reply:
I'm worried about a couple things. First, saying we "support LibSass" when we actually only support a [two-and-a-half-year-old beta release](https://github.com/sass/libsass/releases?after=3.3.0) is pretty misleading, especially when we *will* be supporting up-to-date versions of Dart Sass. Second, I'm worried that if people start relying on these implementations being swappable and that property breaks later on (which it may very well), we'll have unexpected and painful breakage for downstream users.

Reply:
Removed all references to libsass/sassc in docstrings